website/content/docs/connect/ca/vault.mdx

```diff
@@ -181,122 +163,171 @@ The configuration options are listed below.
 
 @include 'http_api_connect_ca_common_options.mdx'
 
-## Root and Intermediate PKI Paths
+### Root and Intermediate PKI Paths
 
 The Vault CA provider uses two separately configured
 [PKI secrets engines](https://www.vaultproject.io/docs/secrets/pki)
-for managing Connect certificates.
+for managing Consul service mesh certificates.
 
-The `RootPKIPath` is the PKI engine for the root certificate. Consul will
-use this root certificate to sign the intermediate certificate. Consul will
-never attempt to write or modify any data within the root PKI path.
+The `RootPKIPath` is the PKI engine for the root certificate. Consul uses this root certificate to sign the intermediate certificate. Consul does not attempt to write or modify any data within the root PKI path.
 
 The `IntermediatePKIPath` is the PKI engine used for storing the intermediate
-signed with the root certificate. The intermediate is used to sign all leaf
-certificates and Consul may periodically generate new intermediates for
+signed with the root certificate. The intermediate signs all leaf
+certificates, and Consul may periodically generate new intermediates for
 automatic rotation. Therefore, Consul requires write access to this path.
 
-If either path does not exist, then Consul will attempt to mount and
-initialize it. This requires additional privileges by the Vault token in use.
-If the paths already exist, Consul will use them as configured.
+If neither path exists, then Consul attempts to mount and
```

Comment:
The original statement wasn't clear enough. The suggestion replacement isn't logically equivalent, though. I'll need to update.